File: resources/base/client/ai.lua
local multiplier, randomcops = Config.ai.multiplier, Config.ai.randomcops

Citizen.CreateThread(
    function()
        while true do
            Citizen.Wait(0)
            SetAmbientVehicleRangeMultiplierThisFrame(multiplier)
            SetCreateRandomCops(randomcops)
            SetCreateRandomCopsNotOnScenarios(randomcops)
            SetCreateRandomCopsOnScenarios(randomcops)
            SetGarbageTrucks(Config.ai.garbagetrucks)
            SetParkedVehicleDensityMultiplierThisFrame(multiplier)
            SetPedDensityMultiplierThisFrame(multiplier)
            SetRandomBoats(Config.ai.randomboats)
            SetRandomVehicleDensityMultiplierThisFrame(multiplier)
            SetScenarioPedDensityMultiplierThisFrame(multiplier, multiplier)
            SetVehicleDensityMultiplierThisFrame(multiplier)

            local player, ped = PlayerId(), PlayerPedId()
            local x, y, z = table.unpack(vector3(GetEntityCoords(ped)))

            ClearAreaOfCops(x, y, z, 100, 0)
            ClearPlayerWantedLevel(player)
            SetPoliceIgnorePlayer(player, randomcops)
            SetPoliceRadarBlips(randomcops)
        end
    end
)

File: turbo/src/resources/base/client/ai.ts
const { garbagetrucks, multiplier, randomboats, randomcops } = BaseConfig["ai"],
  multiplier_funcs = [
    SetAmbientVehicleRangeMultiplierThisFrame,
    SetParkedVehicleDensityMultiplierThisFrame,
    SetPedDensityMultiplierThisFrame,
    SetRandomVehicleDensityMultiplierThisFrame,
    SetScenarioPedDensityMultiplierThisFrame,
    SetVehicleDensityMultiplierThisFrame,
  ];

setTick(async function () {
  while (true) {
    await CWait(1000);
    SetGarbageTrucks(garbagetrucks);
    for (const key in multiplier_funcs)
      multiplier_funcs[key](multiplier, multiplier);
    SetRandomBoats(randomboats);

    const player = PlayerId(),
      [x, y, z] = GetEntityCoords(PlayerPedId());

    ClearAreaOfCops(x, y, z, 100, 0);
    ClearPlayerWantedLevel(player);
    SetPoliceIgnorePlayer(player, randomcops);
    SetPoliceRadarBlips(randomcops);
  }
});
